cosmos/dbt/selector.py

```diff
@@ -85,6 +89,9 @@ def select_nodes_ids_by_intersection(nodes: dict[str, DbtNode], config: Selector
     """
     selected_nodes = set()
     for node_id, node in nodes.items():
+        if config.is_empty_config:
+            continue
+
```

Comment:
We don't need to iterate over all nodes and continue since the config is the same for all of them:

```suggestion
    if not config.is_empty_config:    
        for node_id, node in nodes.items():
```
This change will also require the rest of the "for" block to be indented.